client/src/modules/report-group/index.html

```diff
@@ -0,0 +1,170 @@
+                        </div>
+                    </div>
+                    <!-- description -->
+
+                    <div
+                        class="form-group"
+                        ng-class="{'has-error' : ReportGroupRegistrationForm.description.$invalid && ReportGroupRegistrationForm.$submitted}">
+
+                        <label class="control-label ng-binding" translate>
+                          FORM.LABELS.DESCRIPTION
+                        </label>
+
+                        <div class="input-group col-lg-12">
+                            <textarea 
+                                class="form-control"
+                                name="description"
+                                ng-model="ReportGroupCtrl.reportGroup.description"
+                                autocomplete="off"
+                                required>
+                            </textarea>
+                            <div class="help-block"
+                                ng-messages="ReportGroupRegistrationForm.description.$error"
+                                ng-show="ReportGroupRegistrationForm.$submitted">
+                                  <div ng-messages-include="modules/templates/messages.tmpl.html"></div>
+                            </div>
+                        </div>
+                    </div>
+
+
+                    <bh-loading-button 
+                        ng-show="ReportGroupCtrl.selectedReportGroup.selected == false"
+                        class="pull-right" loading-state="BedRegistrationForm.$loading">
+                          {{ "FORM.BUTTONS.SAVE" | translate }}
+                    </bh-loading-button>
+
+                    <span 
+                        class="btn btn-danger"
+                        ng-click="ReportGroupCtrl.remove(ReportGroupRegistrationForm)"
+                        ng-show="ReportGroupCtrl.selectedReportGroup.selected == true"
+                        class="pull-right">
+                          {{ "FORM.BUTTONS.DELETE" | translate }}
+                    </span>
+
+                    <bh-loading-button 
+                        ng-show="ReportGroupCtrl.selectedReportGroup.selected == true"
+                        class="pull-right"
+                        loading-state="BedRegistrationForm.$loading">
+                          {{ "FORM.BUTTONS.UPDATE" | translate }}
+                    </bh-loading-button>
+
+                </form>
+
+            </div>
+            <!-- end new Reporting group form -->
+
+            <!-- start grid of Reports group -->
+
+            <div class="col-lg-7">
+                <!--the grid -->
+
+                <div 
+                    id="report-groups-grid"
+                    ui-grid-auto-resize
+                    ui-grid-resize-columns
+                    ui-grid-move-columns
+                    ui-grid-cellNav
+                    ui-grid-grouping
+                    ui-grid-exporter
+                    ui-grid-save-state
```

Comment:
Please make sure you only use the plugins you need - anything extra will slow the page down.